### PR TITLE
Add Ubuntu Mono to Typography Section

### LINKED ISSUE #5: Add code editor typeface to style guide
The font is Ubuntu mono. Here's the full information:

sharing_frame_window

### PR DESCRIPTION
**Pull request by user512**
* Add `ubuntu_mono.zip` to `/downloads`

closes https://github.com/FreeCodeCamp/design-style-guide/issues/5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,6 +118,9 @@
 				<br></br>
 				<p>Free Code Camp uses <a href="https://www.dafont.com/saxmono.font">SaxMono</a>, for our logo.</p>
 				<span class="small"><a href="downloads/saxmono.zip">Download SaxMono</a></span>
+				<br></br>
+				<p>Free Code Camp uses <a href="https://www.google.com/fonts/specimen/Ubuntu+Mono">Ubuntu Mono</a> in code editor.</p>
```
Comment (ghost) [inline index.html]:
```diff
- in code editor.
+ for our code editor.
```

File: index.html [hunk 2]
```diff
@@ -118,6 +118,9 @@
...
+				<span class="small"><a href="downloads/ubuntu_mono.zip">Download Ubuntu Mono</a></span>
```
Comment (ghost) [inline index.html:123]: Can you rename the other items in the downloads folder to use underscores for consistency? For example; `sax_mono.zip`. Thanks 😃

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by user512** [comment]
@atjonathan ready to merge?

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
### LGTM 👍 
Thanks @user512 for contributing!

**Comment by QuincyLarson** [comment]
@user512 thanks!

**Pull request merged**